playground/update.sh

```diff
@@ -1,24 +1,39 @@
-#! /bin/sh
+#!/bin/bash
 set -e
 
 go install github.com/gopherjs/gopherjs/...
 
 go generate github.com/gopherjs/gopherjs.github.io/playground/internal/imports
 
-gopherjs install -m github.com/gopherjs/gopherjs.github.io/playground github.com/gopherjs/gopherjs/js
-cp $GOPATH/bin/playground.js $GOPATH/src/github.com/gopherjs/gopherjs.github.io/playground/playground.js
-cp $GOPATH/bin/playground.js.map $GOPATH/src/github.com/gopherjs/gopherjs.github.io/playground/playground.js.map
+# Build playground itself.
+gopherjs build -m
 
-PKG=$GOPATH/src/github.com/gopherjs/gopherjs.github.io/playground/pkg
+# The GOPATH workspace where the GopherJS project is.
+GOPHERJSGOPATH=`go list -f '{{.Root}}' github.com/gopherjs/gopherjs`
```

Comment:
Sounds good, I'll do that. I didn't know what the difference between two was, but I went with backticks because I've used them more in the past.